src/library/scala/collection/BitSetLike.scala

```diff
@@ -216,6 +216,19 @@ trait BitSetLike[+This <: BitSetLike[This] with SortedSet[Int]] extends SortedSe
     throw new NoSuchElementException("Empty BitSet")
   }
 
+  /** Returns the first positive integer not in the BitSet
+   */
+  def nonHead: Int = {
```

Comment:
I don't think this method is of enough general interest to be added to the standard library. I guess the issue is that it cannot be efficiently implemented outside the collection?